packages/esm-patient-attachments-app/src/camera-media-uploader/camera-media-uploader.scss

```diff
@@ -9,7 +9,8 @@
 
 :global(.react-html5-camera-photo) {
```

Comment:
Can we please wrap this global rule inside of a class so it doesn't accidentally affect other microfrontends? I believe global updates to package classes should be located globally - in esm-styleguide

Reply:
Sure!